scripts/linux/publishReleasePackages.sh

```diff
@@ -239,7 +235,11 @@ publish_to_github()
 
         #Create Release Page
         url="https://api.github.com/repos/Azure/azure-iotedge/releases"
-        body=$(jq -n --arg version "$VERSION" --arg body "$(cat $WDIR/content.txt)" '{tag_name: $version, name: $version, target_commitish:"main", body: $body}')
+        reqBody='{tag_name: $version, name: $version, target_commitish:"main", draft: true, body: $body}'
+        if [[ $SKIP_UPLOAD == "false" ]]; then
+            reqBody='{tag_name: $version, name: $version, target_commitish:"main", body: $body}'
+        fi
+        body=$(jq -n --arg version "$VERSION" --arg body "$(cat $WDIR/content.txt)" "$reqBody")
         sudo rm -rf $WDIR/content.txt
```

Comment:
I know this isn't the main change in the PR. But on line 230 it would help readability imo if it also explains what the content structure is. Not necessarily very detailed, but a high level description.

Reply:
https://docs.github.com/en/rest/releases/releases

Reply:
I looked at this doc and it is still not clear to me. When I was reading this there is a lot of cognitive load needed to understand what is in content.txt, and I gave up. Still suggest a comment explaining the structure. Don't understand the existing comment.

Reply:
It also isn't that api that line 30 has to do with, it is this one right?
https://api.github.com/repos/Azure/iotedge/contents

My high level point is that reading through all this bash, I'd prefer more places/ways where I can look at a big block of implementation and abstract it into an idea. And not have to dive into multiple variables, functions, api references, bash syntax, etc. When bash gets complicated for me I often see this happen because syntax around function returns is unwieldy. So comments are the best we can do I think.

Reply:
Added the function level header to the `publishReleasePackages.sh`.